dotnet/src/IntegrationTests/Connectors/Memory/Weaviate/WeaviateVectorStoreRecordCollectionTests.cs

```diff
@@ -470,7 +470,6 @@ private VectorStoreRecordDefinition GetTestHotelRecordDefinition()
                 new VectorStoreRecordDataProperty("Description", typeof(string)),
                 new VectorStoreRecordDataProperty("Timestamp", typeof(DateTimeOffset)),
                 new VectorStoreRecordVectorProperty("DescriptionEmbedding", typeof(ReadOnlyMemory<float>?), 4) {IndexKind = IndexKind.Hnsw, DistanceFunction = DistanceFunction.CosineDistance }
-
```

Comment:
It seems that the .NET 8 build complains about lack of space before `IndexKind` in the line above (I can't repro it locally, just guessing)

```diff
- new VectorStoreRecordVectorProperty("DescriptionEmbedding", typeof(ReadOnlyMemory<float>?), 4) {IndexKind = IndexKind.Hnsw, DistanceFunction = DistanceFunction.CosineDistance }
+ new VectorStoreRecordVectorProperty("DescriptionEmbedding", typeof(ReadOnlyMemory<float>?), 4) { IndexKind = IndexKind.Hnsw, DistanceFunction = DistanceFunction.CosineDistance }
```

Reply:
Thank you @adamsitnik!